integration/test/basic/basic_test.go

```diff
@@ -26,10 +25,10 @@ func TestChartInstalled(t *testing.T) {
 	operation := func() error {
 		rc, err = helmClient.GetReleaseContent("tb-release")
 		if err != nil {
-			return fmt.Errorf("could not retrieve release content: %v", err)
+			return microerror.Newf("could not retrieve release content: %v", err)
```

Comment:
`Maskf` is probably what you want. This here creates a new error and breaks the trace.

Reply:
Sure, done.